server.php

```diff
@@ -7,15 +7,13 @@
  * @author   Taylor Otwell <taylor@laravel.com>
  */
 
-$uri = urldecode(
-    parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)
-);
+$uri = urldecode(parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH));
```

Comment:
Same as my comments with `bootstrap/`, this file seems to be part of the framework and not the app, so not sure about changing its format and the possible impact on upgrade.